themes/kiali/layouts/partials/home/video-getting-started-section.html

```diff
@@ -10,6 +10,7 @@ <h1>
     </h1>
 
     <div class="embed-video-links">
+        <iframe class="video-iframe-links" src="https://www.youtube.com/embed/ht9M_GW7k6c"></iframe>
```

Comment:
Not a blocker, but not sure if we want to limit these two three videos to avoid wrap on the website.

Reply:
Ok good idea. I'm doing one more video. I will incorporate that for that video.